appium/webdriver/extensions/flutter/flutter_commands.py

```diff
@@ -0,0 +1,166 @@
+            offset: The x and y offsets from the element to perform the long press at. If not specified, the long press is performed at the element's center.
+            
+        Returns:
+            None:
+        """
+        opts = {'origin': element}
+        if offset:
+            opts['offset'] = {'x': offset[0], 'y': offset[1]}
+        self.execute_flutter_command('longPress', opts)
+            
+    def perform_drag_and_drop(self, source: WebElement, target: WebElement) -> None:
+        """
+        Performs a drag-and-drop operation from a source element to a target element.
+
+        Args:
+            source: The element to drag from.
+            target: The element to drop onto.
+            
+        Returns:
+            None:
+        """
+        self.execute_flutter_command('dragAndDrop', {'source': source, 'target': target})
+        
+    def scroll_till_visible(self, scroll_to: FlutterFinder, scroll_direction: Optional[str] = 'down', **opts: Any) -> WebElement:
+        """
+        Scrolls until the specified element becomes visible.
+
+        Args:
+            scroll_to: The Flutter element to scroll to.
+            scroll_direction: The direction to scroll up/down. Defaults to 'down'.
+                
+        KeywordArgs:
+                scrollView (str): The view of the scroll.
+                delta (int): delta for the scroll
+                maxScrolls (int): Max times to scroll
+                settleBetweenScrollsTimeout (int): settle timeout
+                dragDuration (int): time gap between each scroll
+        
+        Returns:
+            Webelement: scrolled element
+        """
+        opts['finder'] = scroll_to.to_dict()
+        opts['scrollDirection'] = scroll_direction
+        return self.execute_flutter_command('scrollTillVisible', opts)
+        
+    def inject_mock_image(self, value: str) -> str:
+        """
+        Injects a mock image to the device. The input can be a file path or a base64-encoded string.
+
+        Args:
+            value: The file path of the image or a base64-encoded string.
+            
+        Returns:
+            str: Image ID of the injected image.
+        """
+        import os
+        if os.path.isfile(value):        
+            with open(value, "rb") as image_file:
+                base64_encoded_image = base64.b64encode(image_file.read()).decode('utf-8')
+        else:
+            base64_encoded_image = value
+        return self.execute_flutter_command('injectImage', {'base64Image': base64_encoded_image})
+    
+    def activate_injected_image(self, image_id: str) -> None:
+        """
+        Activates an injected image with image ID.
+
+        Args:
+            image_id: The ID of the injected image to activate.
+
+        Returns:
+            None:
+        """
+        self.execute_flutter_command("activateInjectedImage", {'imageId': image_id})
+        
+    def execute_flutter_command(self, scriptName: str, params: dict) -> Any:
```

Comment:
Is this a file private method?

```suggestion
    def _execute_flutter_command(self, scriptName: str, params: dict) -> Any:
```

Reply:
No I dnt want it as a private method. I have added the documentation.